packages/gatsby-remark-code-repls/package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "gatsby-remark-code-repls",
   "description": "Gatsby plugin to auto-generate links to popular REPLs like Babel and Codepen",
-  "version": "2.0.8",
+  "version": "3.0.0",
```

We will make sure we publish a major :)
```suggestion
  "version": "2.0.8",
```